### PR TITLE
Historical Norse Names for Vrykul

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Replaced Vrykul names with ones that are partially based on WoW vrykul names and Nordiskt runnamnslexikon by Lena Peterson and other historical sources.

### CODE REVIEW

File: common/cultures/culture_vrykul.txt [hunk 1]
```diff
@@ -10,24 +10,18 @@ vrykul_group = {
...
-			Hjuyrs Hont Hovoesviw Hrarnemarn Hrefarn Hreldgutord Hrund Hrurjuva Hruyrs Ildgu Imdorn Irbrarhurn Iredrag Kar Kefoeldid Keseld Kork Milment Reng Roldgek Sithidril Skamanin Skel Skirn Skothihirn Slelskar Slithinurn Sordiw
-			Slod Slullskinnull Slundell Jannuyrdrom Jannvekke Jig Jillskog Jyrr Kanilfr Kerlerdr Kirn Koemderr Kog Kork Kulro Migg Minog Mirkamak Mognvuthom Murk Olskannerdr Othemill Rasgrar Rennlag Reylorhegg Rilren Rordun Rud Rulmifi
-			Skaad Skaktuk Skeri Skim Skoethang Skong Skor Skurvoll Slannvor Sleind Slilbreld Slinvent Slugegnha Slund Slunnvurs Slurdruvogg Stand Stil Stirholgog Stulf Sugnvofo Sval Svelburn Tek Thokuw Thuktern Thundellelfr Tulbrurhund
-			Tundunnus Ukegg Vinnirdr Votathelfr Vurve Yant Yerdr Yilf Yow Slusek Stardurn Stogazand Storhumdold Stud Stukto Stylstagoll Svelsto Sving Tant Teilfr Tell Ters Therlared Thethnos Thiljing Thillondin Ugalgew Ulbung Uthodhal Uzenvaw
-			Vaaceno Vodreg Vugnhumoll Vuzadhull Yord Yow Yurd
+			Alf Algar Andjor Angantyr Angvold Arni Bagge Balagarde Banahogg Bjorn Brandolf Brinvulf Brulf Brvet Brytag Calder Dagr Drengr Durnolf Dyre Egil Erling
```
Comment (zumbak04) [inline common/cultures/culture_vrykul.txt:13]: Ooof, there are fewer than before.
Me not like.

File: common/cultures/culture_vrykul.txt [hunk 2]
```diff
@@ -10,24 +10,18 @@ vrykul_group = {
```
Comment (zumbak04) [inline common/cultures/culture_vrykul.txt:10]: History vrykul still use old names.
<details>
<summary>Click to expand</summary>

image

</details>
Also, the branch is super outdated.

File: common/cultures/culture_vrykul.txt (hunk 1 above)
Comment (sejtalk) [reply common/cultures/culture_vrykul.txt:13]: There is less of them in general but they are all fitting while the old ones are almost all wrong

**Comment by zumbak04** [comment]
>Bergdottir
>Rovadottir

@LordSilvermane don't use patronymics in the names because they're a thing in CK2.
Otherwise, some names may sound like Rovadottir Rovadottir.

<details>
<summary>Click to expand</summary>

image

</details>

File: common/cultures/culture_vrykul.txt (hunk 1 above)
Comment (zumbak04) [reply common/cultures/culture_vrykul.txt:13]: > There is less of them in general but they are all fitting while the old ones are almost all wrong

I don't get how it affects the fact there are fewer names.

Comment (sejtalk) [reply common/cultures/culture_vrykul.txt:13]: Whats wrong with having fewer names?

Comment (zumbak04) [reply common/cultures/culture_vrykul.txt:13]: > Whats wrong with having fewer names?

What is wrong with having as you said wrong names?

Comment (zumbak04) [reply common/cultures/culture_vrykul.txt:13]: Everyone including me wants more names because it increases diversity.

Comment (sejtalk) [reply common/cultures/culture_vrykul.txt:13]: So it's ok if i copy human names and put them in vrykul namelist?

**Comment by LordSilvermane** [comment]
> don't use patronymics in the names because they're a thing in CK2.
> Otherwise, some names may sound like Rovadottir Rovadottir.

@zumbak04 Fair. Removed them.

**Comment by sejtalk** [comment]
> Composed with help of @geffe1 to add some more genuine scandinavian names in order to bring the total number of names back up a bit.
> 
> Male Names
> Askeladden, Berurdr, Bjall, Borg, Braagard, Brun, Brunvulf, Canute, Deluskand, Eskil, Fidald, Fjorr, Garmr, Gjarn, Gudbrand, Hrafn, Hrund, Isvakt, Ivar, Kork, Mjod, Rennlag, Rordun, Skaad, Skal, Skeri, Skirn, Skor, Styrbjorn, Sval, Tor, Viken
> 
> Female Names
> Afasta, Alfhild, Alfin, Alva, Annalda, Ask, Bolla, Brenna, Dana, Daga, Edda, Edelle, Eira, Elin, Elsa, Embla, Frig, Gothilda, Gudrund, Hidrun, Hilda, Hilldur, Hjort, Holmfreid, Jorunn, Kalda, Karin, Katette, Linn, Liv, Maud, Mina, Olfried, Reida, Ronja, Runa, Saga, Sifrid, Siv, Skalm, Solveig, Tilda, Tora, Torun, Torgun, Tove, Tyra, Undis, Vildi

So after doing some research i modified the male names you suggested to this list of names: Oskeladd, Bjorg, Brunn, Eskill, Fjorr, Garmr, Guthbrandr, Hrafn, Ivar, Skor, Styrbjorn

**Comment by LordSilvermane** [comment]
> So after doing some research i modified the male names you suggested to this list of names: Oskeladd, Bjorg, Brunn, Eskill, Fjorr, Garmr, Guthbrandr, Hrafn, Ivar, Skor, Styrbjorn

Cait would know better than I would, but those names look good to me. I particularly like Oskeladd, Askeladden was my suggestion, but I like yours better. I do think you need a few more to get the name list back to its former levels, though. What about the female names?

**Comment by sejtalk** [comment]
> Cait would know better than I would, but those names look good to me. I particularly like Oskeladd, Askeladden was my suggestion, but I like yours better. I do think you need a few more to get the name list back to its former levels, though. What about the female names?
I will do them later but most likely they will also to much smaller number

**Comment by sejtalk** [comment]
> > Composed with help of @geffe1 to add some more genuine scandinavian names in order to bring the total number of names back up a bit.
> > Male Names
> > Askeladden, Berurdr, Bjall, Borg, Braagard, Brun, Brunvulf, Canute, Deluskand, Eskil, Fidald, Fjorr, Garmr, Gjarn, Gudbrand, Hrafn, Hrund, Isvakt, Ivar, Kork, Mjod, Rennlag, Rordun, Skaad, Skal, Skeri, Skirn, Skor, Styrbjorn, Sval, Tor, Viken
> > Female Names
> > Afasta, Alfhild, Alfin, Alva, Annalda, Ask, Bolla, Brenna, Dana, Daga, Edda, Edelle, Eira, Elin, Elsa, Embla, Frig, Gothilda, Gudrund, Hidrun, Hilda, Hilldur, Hjort, Holmfreid, Jorunn, Kalda, Karin, Katette, Linn, Liv, Maud, Mina, Olfried, Reida, Ronja, Runa, Saga, Sifrid, Siv, Skalm, Solveig, Tilda, Tora, Torun, Torgun, Tove, Tyra, Undis, Vildi
> 
> So after doing some research i modified the male names you suggested to this list of names: Oskeladd, Bjorg, Brunn, Eskill, Fjorr, Garmr, Guthbrandr, Hrafn, Ivar, Skor, Styrbjorn

Same for female names:
Alfhildr, Alva, Bothildr, Dana, Daga, Edda, Elin, Embla, Gauthildr, Guthrun, Hildr, Kalda, Saga, Solveig, Thorunnr, Thyrvi

**Comment by LordSilvermane** [comment]
The new names that were recently added look good. The preset characters who hold counties (and their immediate families) still have the old names, which needs to be looked at, as it is quite jarring to see the difference. Finally, although I am glad there are new names, there still is a considerable deficit. I tried generating commanders and within five names I got two with the same name. Looking at random characters in court, I saw several names frequently repeating, and in one case, there were two people in the same court with the same name. The new names are very, very good (especially when compared to the old ones, ugh), but I do think more need to be added.

File: history/characters/100000_vrykul.txt [hunk 3]
```diff
@@ -62,7 +62,7 @@
 
 # dynasty=70002
 100200={
-	name=Skeld
+	name=Ketilld
```
Comment (zumbak04) [inline history/characters/100000_vrykul.txt:65]: A lore character.
https://wow.gamepedia.com/Skeld_Drakeson

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**